.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: c
+language: python
```

Comment:
Oops :(
But we can't test multiple version such as 2.7 and 3.5 without using `language: python`.